src/properties.cpp

```diff
@@ -1008,6 +2458,15 @@ namespace Exiv2 {
         { "camerajpeg",       N_("Camera JPEG")                              }
     };
 
+    //! XMP iptcExt:DigitalSourceType
```

Comment:
Could you please add a comment with this link for reference?
```
https://cv.iptc.org/newscodes/digitalsourcetype/
```

Reply:
I have added this.